.github/workflows/zeebe-benchmark.yml

```diff
@@ -117,6 +127,74 @@ jobs:
           build-args: DIST=build
           provenance: false
           target: app
+  build-operate-image:
+    name: Build + Push Operate Docker image
+    if: ${{ inputs.operate-tag != '' }}
```

Comment:
ℹ️ I simply copied the build part from the other workflow, and adjusted the credentials.

I guess there might be more improvements possible but I wanted to keep the changes small for now.